app/repositories/package.scala

```diff
@@ -195,7 +195,7 @@ package object repositories {
     val preferredName = personalDetailsRoot.getAs[String]("preferredName").get
 
     val assistanceDetailsRoot = doc.getAs[BSONDocument]("assistance-details").get
-    val guaranteedInterview = assistanceDetailsRoot.getAs[String]("guaranteedInterview").contains("Yes")
+    val guaranteedInterview = assistanceDetailsRoot getAs[Boolean]("guaranteedInterview") getOrElse false
```

Comment:
I mean, this clearly works, but we use . notation every where else, did you just fancy a change?